packages/participants-svc/jest.config.js

```diff
@@ -13,12 +13,12 @@ module.exports = {
     coveragePathIgnorePatterns: ["./src/tmp_files"],
     coverageReporters: ["text", ["json", {file: `../../../coverage/${packageName}-final.json`}]],
     clearMocks: true,
-    coverageThreshold: {
+    /* coverageThreshold: {
```

Comment:
Can you please add a comment here as a TODO to enable it soon after the initial checks are done of this merge..

Reply:
Just added a TODO comment, Sam.